menu/additional_apps.sh

```diff
@@ -17,17 +17,25 @@ debug_mode
 # Must be root
 root_check
 
+# Set the startup switch
+if [ -f "$SCRIPTS/nextcloud-startup-script.sh" ]
+then
+    STARTUP_SWITCH="ON"
+else
+    STARTUP_SWITCH="OFF"
+fi
```

Comment:
I mean we could theoratically put that into the library...

Reply:
No, since we can't rely on that it will get fetched each time during the startup script. This implementation is better.